lib/parallel_tests/cli.rb

```diff
@@ -35,6 +40,9 @@ def execute_in_parallel(items, num_processes, options)
             puts
           end
           reprint_output(result, lock.path) if options[:serialize_stdout]
+          if options[:fail_fast] && result.fetch(:exit_status) != 0
```

Comment:
I originally did that but found it was only killing direct children of the process, not grandchildren. Maybe this would be a good update for the Parallel gem? I can look into that.